packages/react-drag-drop/src/next/components/DragDrop/DraggableDataListItem.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+import dragStyles from '@patternfly/react-styles/css/components/DragDrop/drag-drop';
+import { DragButton } from './DragButton';
+import { DataListItemRow, DataListControl } from '@patternfly/react-core';
+
+export interface DraggableDataListItemObject {
+  id?: string;
+  content?: React.ReactNode;
+}
+
+export interface DraggableDataListItemProps extends React.HTMLProps<HTMLLIElement> {
+  /** Content rendered inside DragDrop */
+  children?: React.ReactNode;
+  /** Don't wrap the component in a div. Requires passing a single child. */
```

Comment:
Maybe "Flag indicating that a div should not wrap the componet". Or something along those lines.

Reply:
This prop is unused currently so removing for now. Might have been a holdover from a previous iteration.